python/tvm_ffi/testing.py

```diff
@@ -90,5 +108,5 @@ class _TestCxxClassDerived(_TestCxxClassBase):
 
 @c_class("testing.TestCxxClassDerivedDerived")
 class _TestCxxClassDerivedDerived(_TestCxxClassDerived):
-    v_str: str = field(default_factory=lambda: "default")
-    v_bool: bool
+    v_str: str = field(default_factory=lambda: "default")  # type: ignore[assignment]
```

Comment:
TODO: need to figure out why `field(...)` triggers a mypy complain, despite `field_specifiers` are used properly